SingularityBase/src/main/java/com/hubspot/singularity/executor/SingularityExecutorLogrotateFrequency.java

```diff
@@ -0,0 +1,19 @@
+package com.hubspot.singularity.executor;
+
+public enum SingularityExecutorLogrotateFrequency {
+    HOURLY("monthly"),
```

Comment:
Right, forgot it doesn't actually have an hourly setting. Would it make sense to instead write something to `/etc/cron.hourly/`? Value probably doesn't matter in either case since it's manually triggered. Then we could let cron do it's thing rather than having the executor worry about it. Just a thought, this way would work too.